.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,40 @@
+name: NPM package publishing
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish_npm:
+    name: Publish ${{ matrix.package }} NPM package
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package: [javy, javy-cli]
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Install package dependencies
+        run: npm install
+        working-directory: npm/${{ matrix.package }}
+
+      - name: Build NPM package
+        if: matrix.package == 'javy'
```

Comment:
This conditional is here because the `javy` NPM package has a build step that needs to be run while `javy-cli` does not have a build step.